src/CentreonRemote/Domain/Service/ConfigurationWizard/ServerConnectionConfigurationService.php

```diff
@@ -138,8 +138,10 @@ protected function insertConfigResources($serverID)
             throw new \Exception('Resources records from `cfg_resource` could not be fetched.');
         }
 
-        if ($results[0]->resource_name != '$USER1$' ||
-            $results[1]->resource_name != '$CENTREONPLUGINS$'
+        if (
+            !in_array($results[0]->resource_name, ["\$CENTREONPLUGINS$","\$USER1$"])
+            && !in_array($results[1]->resource_name, ["\$CENTREONPLUGINS$,","\$USER1$"])
```

Comment:
you should update sql query instead of updated this condition

Reply:
Changed Sql statement